### ISSUE TITLE
noqa is not respected for E241

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#106]
In GitLab by @gforcada on Jun 3, 2015, 08:53

If you have something like this:

```
dictionary = {
    'key1'     : 'value1',  # noqa
    'long_key2': 'value2',
}
```

The ``# noqa`` comment is ignored and a E241 is reported.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jun 3, 2015, 09:04

`# noqa` is explicitly only supported for [controversial rules][]. This is not a controversial rule, therefore it is not supported.

[controversial rules]: https://pep8.readthedocs.org/en/latest/intro.html?highlight=noqa#error-codes

If you'd like to update our docs at flake8.rtfd.org to point that out, I would happily and speedily accept a merge request updating the docs.
